components/home/header/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from "react-bootstrap";
+import { HeaderImage } from "./HeaderImage";
+import styles from "./Header.module.scss";
+
+interface HeaderProps {
+  tagline: string;
+}
+
+export const Header = ({ tagline }: HeaderProps) => (
+  <header
+    className={`d-flex justify-content-evenly ${styles.header}`}
+    data-testid="header"
+  >
+    <div>
+      <h2>{tagline}</h2>
+      <p className="outline">
+        Grouparoo is an open source framework that helps you move data between
+        your data warehouse and all of your cloud-based tools.
+      </p>
+      <Button type="button" variant="primary" className="rounded-pill">
```

Comment:
Missing a link?  It behaves differently on click too, but maybe that's just from not having anywhere to go!